proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10532,6 +10532,15 @@ function bufferRetype( test )
   test.shouldThrowErrorSync( () => _.bufferRetype( new BufferRaw( 5 ) ), I8x, 'extra' );
 }
 
+function bufferRetypeWithOffset( test )
+{
```

Comment:
Please, extend test routine. 
Needs cases with part of original `raw` buffer from some index to end
```js
var src = new U8x( raw, 6 );
```
And use cases with length
```js
var src = new F32x( raw, 6, 3 );
```